wgpu/backends/wgpu_native/_api.py

```diff
@@ -37,6 +37,8 @@
     SafeLibCalls,
 )
 
+WGPU_WHOLE_SIZE = 0xFFFF_FFFF_FFFF_FFFF  # 2**64 - 1
+WGPU_QUERY_SET_INDEX_UNDEFINED = 0xFFFF_FFFF
```

Comment:
These should be taken from the `lib.WGPU_WHOLE_SIZE`. That way we don't have to worry about these values changing (it has happened).

Reply:
I do not see anywhere else in _api.py or _classes.py where a `#define`d constant is pulled from webgpu.h.  Is there documentation somewhere on how to do so, or at least to ensure that the values are the same in both places? 

Aside from this, are you happy with the push? Should I keep the changes to `set_vertex_buffer`, `set_index_buffer` and `clear_buffer`?  Tests will stay, no matter what.

Reply:
If you search for `lib.` in the source, you'll see some. Could be that its only enum values so far, but `#define`'d constants are there too. The header file is parsed by ffi, producing the `lib` object.

I am happy about the other changes 👍

Reply:
I initially misunderstood what you were asking me to do.  I've made the easy fix rather than the hard fix.